### DISCUSSION TITLE
Remove obsolete golden images

### DISCUSSION THREAD

**Discussion by luizgrp** [cashapp/paparazzi#506] [Ideas]
It would be nice if the library could take care of removing obsolete golden images.

For example, when a function name is renamed, the task `recordPaparazziDebug` generates new golden images for the new function name, leaving the golden images for the previous function name in the folder, which are no longer in use.

**Comment by geoff-powell** [comment]
yes we added a `cleanRecordPaparazzi{Variant}` task to do this exact thing.
